sos/report/plugins/kubernetes.py

```diff
@@ -209,12 +232,21 @@ def setup(self):
         # other changes the `oc` binary may implement
         if self.path_exists('/etc/origin/master/admin.kubeconfig'):
             self.kube_cmd = 'oc'
+        self.resources.extend([
+            'limitranges',
+            'policies',
+            'resourcequotas',
+            'routes'
+        ])
+        self.global_resources.extend([
+            'projects',
+            'pvs'
+        ])
         super().setup()
 
 
 class UbuntuKubernetes(Kubernetes, UbuntuPlugin, DebianPlugin):
 
-    packages = ('kubernetes',)
```

Comment:
I think we need to keep this, don't we?

Reply:
There is no kubernetes package in Canonical k8s, which is deployed with snap.
I've put it back anyway.
Let me know if I need to remove it.

Reply:
We also discover if it is installed as a snap with this method, so this should still be here.